docker/run_jakartaeetck.sh

```diff
@@ -409,7 +409,7 @@ ant config.ri
 cd ${CTS_HOME}
 export PORT=5858
 ${CTS_HOME}/ri/${GF_RI_TOPLEVEL_DIR}/glassfish/bin/asadmin --user admin --passwordfile ${ADMIN_PASSWORD_FILE} -p ${PORT} stop-domain
-${CTS_HOME}/ri/${GF_RI_TOPLEVEL_DIR}/glassfish/bin/asadmin $DISABLE_RESOURCE_VALIDATION --user admin --passwordfile ${ADMIN_PASSWORD_FILE} -p ${PORT} start-domain
+${CTS_HOME}/ri/${GF_RI_TOPLEVEL_DIR}/glassfish/bin/asadmin --user admin --passwordfile ${ADMIN_PASSWORD_FILE} -p ${PORT} start-domain
```

Comment:
This won't set it on GlassFish I will need to take a look at the TCK source and think of how to do this.

Reply:
The above line is reverting the previous change.  

Would ```glassfish/bin/asadmin create-jvm-options -Ddeployment.resource.validation=false``` work?  That is how we set other options, like ```-Dosgi.shell.telnet.port=6667```

Reply:
https://eclipse-ee4j.github.io/glassfish/docs/5.1.0/administration-guide/jvm.html#ggnes describes the above which is what I used in this change.

Reply:
Yes that would work.